src/qit/base/runtime/distributedcontext.py

```diff
@@ -5,15 +5,21 @@
 import itertools
 import logging
 import socket
-
-import cloudpickle
-import distributed
 import resource
-from distributed import Scheduler, Nanny as Worker, Executor, as_completed
-from tornado.ioloop import IOLoop
 
+from qit.base.exception import QitException
 from qit.base.iterator import NoValue
 
+try:
+    import cloudpickle
+    import distributed
+    from distributed import Scheduler, Nanny as Worker, Executor, as_completed
+    from tornado.ioloop import IOLoop
+
+    DistributedImportError = None
```

Comment:
Sorry I don't understand, should they be named with lower_case or not? NoValue in iterator is in CamelCase too.